packages/space-opera/src/test/camera_settings/camera_settings_test.ts

```diff
@@ -65,7 +77,8 @@ describe('camera constraints test', () => {
     const orbit = {phiDeg: 12, thetaDeg: 34, radius: 56};
     reduxStore.dispatch(dispatchInitialOrbit(orbit));
     await cameraSettings.updateComplete;
-    await cameraSettings.cameraOrbitEditor!.updateComplete;
+    // Commented out for flakiness. Kept in case this test begins to fail.
+    // await cameraSettings.cameraOrbitEditor!.updateComplete;
```

Comment:
removing this update, fixed this test. I just wanted to keep the comment of it here just in case this test begins to fail in the future

Reply:
I don't totally understand the comment. Was it flaky with or without this line? How might this line help?

Reply:
Yeah, that comment is confusing. I'll just remove it. It was basically saying, we don't need that updateComplete anymore.